app/views/cases/shared/_sar_response_address.html.slim

```diff
@@ -0,0 +1,7 @@
+tr.requester-email
```

Comment:
We could also have postal address here.

Reply:
I disagree.  There are only two reply methods enumerated: send by post, send by email, not send by both post and email, therefore the response address should reflect that.

Reply:
Are we never going to display the postal address?